core/locale/main/values/strings_ui.xml

```diff
@@ -384,6 +385,7 @@
     <string name="first_names">First names</string>
     <string name="basic_details">Basic details</string>
     <string name="home_address">Home address</string>
+    <string name="address">Address</string>
```

Comment:
Perhaps we should remove home_address so that we just use address on person detail?